librz/core/cconfig.c

```diff
@@ -3747,7 +3747,7 @@ RZ_API int rz_core_config_init(RzCore *core) {
 		NULL);
 	SETPREF("search.prefix", "hit", "Prefix name in search hits label");
 	SETI("search.maxhits", 0, "Maximum number of hits ('0' means no limit)");
-	SETBPREF("search.show_progress", "true", "Show the search process.");
+	SETPREF("search.show_progress", "1", "Show the search process (true, false, intervals)");
```

Comment:
make this as a option see `str.encoding` for the example..

Reply:
ok b9355bdcf7767ae0ddc442b7dea8fa47a87c742b ... the reason `"1"` is used instead of `"true"` as the initial value is because of what appears to be a config design flaw that limits config options to only boolean values if the initial value is either `"true"` or `"false"`